Cargo.toml

```diff
@@ -20,9 +20,10 @@ authors = ["PRQL Developers"]
 edition = "2021"
 license = "Apache-2.0"
 repository = "https://github.com/PRQL/prql"
-# This isn't tested since `cargo-msrv` doesn't support workspaces; instead we
-# test `metadata.msrv` in `prqlc`
-rust-version = "1.70.0"
+# This isn't tested since `cargo msrv verify` doesn't support workspaces
+# https://github.com/foresterre/cargo-msrv/issues/590
+# But we can find the MSRV by `cargo msrv find`
+rust-version = "1.77.2"
```

Comment:
If no crates have `rust-version.workspace = true`, does this have any effect?

Reply:
What do you mean?
lutra has `rust-version.workspace = true`

Reply:
OK I see. Possible we push that down to lutra, and leave this empty. 

Hopefully we can have an MSRV for most of the workspace set here, and then lutra can have its own higher one. But possibly that won't work yet.

Reply:
Ah great, thanks a lot for following up with that. I added a comment to the issue.

If that issue is resolved, I would vote to have an MSRV of something like 1.73.0 here, have most crates inherit it, and then Lutra has a different higher one.